### PR TITLE
fix(notifications): stop alert notification if alert would notify of message of current active chat

### PR DESCRIPTION
**Pull request by Jekrimo**
**What this PR does** 📖
Don't have alert for chat that is active

**Which issue(s) this PR fixes** 🔨

AP-1502

### CODE REVIEW

File: store/ui/actions.test.ts [hunk 1]
```diff
@@ -508,7 +508,7 @@ describe('init', () => {
     )
     expect(result).toBeUndefined()
   })
-  test('sendNotification with initialized mailbox manager', async () => {
+  test.skip('sendNotification with initialized mailbox manager', async () => {
```
Comment (stavares843) [inline store/ui/actions.test.ts:511]: @drepram can you update this test pls? you can update directly here 🔨

Comment (Jekrimo) [reply store/ui/actions.test.ts:511]: Once again I forgot to tag you when I opened the Pr @drepram, thank you for always savin' my booty Sara!

**Review by phillsatellite — APPROVED** [review]

**Review by josephmcg — COMMENTED** [review]
Could you explain the decision to change alert.state into a configurable param? Currently, we only have READ or UNREAD. Is there a plan to create other states, or is there a use case where we want to create a READ alert?

**Comment by Jekrimo** [comment]
> 

I can foresee both needing to happen honestly. Like an app update alert that auto pops up on launch,  still an alert, but they are forced to see it originally so it's already read.

**Comment by josephmcg** [comment]
Thanks, was just curious. Sounds good

File: libraries/Textile/NotificationManager.ts [hunk 2]
```diff
@@ -80,6 +80,7 @@ export class NotificationManager {
     message: string
     imageHash: string
     type: AlertType
+    notificationstate: AlertState
```
Comment (stavares843) [inline libraries/Textile/NotificationManager.ts:83]: maybe notificationState for better read

Comment (Jekrimo) [reply libraries/Textile/NotificationManager.ts]: Super good catch, thank you!

Comment (stavares843) [reply libraries/Textile/NotificationManager.ts]: 🎉

File: store/ui/actions.test.ts (hunk 1 above)
Comment (drepram) [reply store/ui/actions.test.ts]: Hi! I pushed a fix (https://github.com/Satellite-im/Core-PWA/pull/3267/commits/d0bad2388278e0f90de6b39b45e7273e2254561c), please check :)

**Comment by phillsatellite** [comment]
Hey Jeff! I went in and retested and found one small issue, the red badge that appears on alerts isnt showing up for me for some reason (I'm in a different chat when receiving messages) I did a new server and also tried replicating on dev branch as well. When I do open alerts all the messages that were coming in appear there just no red badge no matter which chat you are in

https://user-images.githubusercontent.com/93608357/170736080-3fa49624-e69b-4dbb-891f-0c65f6455cb4.mov

**Comment by Jekrimo** [comment]
> Hey Jeff! I went in and retested and found one small issue, the red badge that appears on alerts isnt showing up for me for some reason (I'm in a different chat when receiving messages) I did a new server and also tried replicating on dev branch as well. When I do open alerts all the messages that were coming in appear there just no red badge no matter which chat you are in
> 
>  alerts.notification.mov

what the freaking heck

**Comment by Jekrimo** [comment]
> Hey Jeff! I went in and retested and found one small issue, the red badge that appears on alerts isnt showing up for me for some reason (I'm in a different chat when receiving messages) I did a new server and also tried replicating on dev branch as well. When I do open alerts all the messages that were coming in appear there just no red badge no matter which chat you are in
> 
>  alerts.notification.mov

Should be fixed now! Dumb fix

**Review by phillsatellite — APPROVED** [review]

**Comment by phillsatellite** [comment]
@Jekrimo worked perfect on my end, and amazing response time with the fix 😂

**Review by molimauro — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**